jsx-dev-runtime.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @typedef {import('xast').Element} Element
+ * @typedef {import('xast').Root} Root
+ * @typedef {import('./lib/index.js').XChild} XChild
+ * @typedef {import('./lib/runtime.js').JSXProps} JSXProps
+ */
+
+import {jsx} from './jsx-runtime.js'
+
+export const jsxDEV =
+  /**
+   * @type {{
+   *   (name: null|undefined, props: {children?: XChild}, ...unused: unknown[]): Root
+   *   (name: string, props: JSXProps, ...unused: unknown[]): Element
+   * }}
+   */
+  (
+    function (name, props) {
+      return jsx(/** @type {string} */ (name), props)
+    }
+  )
+export {Fragment} from './jsx-runtime.js'
```

Comment:
```suggestion

export {Fragment} from './jsx-runtime.js'
```

Feels a bit cleaner?

Reply:
True. `npm test` autofixed it into this.